flowy/src/lib.rs

```diff
@@ -117,13 +117,13 @@ pub fn generate_config_solar(path: &Path, lat: f64, long: f64) -> Result<(), Box
 /// Generates the config file. Takes the wallpaper folder path as args.
 pub fn generate_config(path: &Path) -> Result<(), Box<dyn Error>> {
     let walls = get_dir(path, "")?;
-    let length = walls.len();
-    let div = 1440 / length;
+    // Offset in seconds for each wallpaper
+    let div = 86400 / walls.len();
     let mut times = Vec::new();
-    let mut start_sec = 0;
-    for _ in 0..length {
-        times.push(format!("{:02}:{:02}", start_sec / 60, start_sec % 60));
-        start_sec += div;
+
+    for i in 0..walls.len() {
+        let offset = div * i;
+        times.push(format!("{:02}:{:02}", offset / 3600, (offset / 60) % 60));
```

Comment:
Will this statement push times in the form ```18:32```?  I just want to know if the program is run in the normal ```--dir``` mode, it still works as expected.

Reply:
Run it on your local and ```cat``` into the ```config.toml```file to see if everything has been generated right.

Reply:
```toml
times = ["00:00", "06:00", "12:00", "18:00"]
walls = ["file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-01.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-02.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-03.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/NIGHT-01.jpeg"]
```
Looks fine to me

Reply:
```toml
times = ["05:48", "10:55", "16:02", "21:09"]
walls = ["file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-01.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-02.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/DAY-03.jpeg", "file:///home/leo/Pictures/Wallpapers/Ghibli Dynamic/NIGHT-01.jpeg"]
```
And this is `--solar` mode (coordinates `45.4668 9.1905`, Milan)